packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -124,144 +126,157 @@ export const render = async (req, res, next) => {
     const includeServerTimingHeader = '__server_timing' in req.query
     const shouldTrackPerformance = includeServerTimingHeader || process.env.SERVER_TIMING
 
-    if (!isServerTracingInitialized() && shouldTrackPerformance) {
-        initializeServerTracing()
-    }
-
-    res.__performanceTimer = new PerformanceTimer({enabled: shouldTrackPerformance})
-    res.__performanceTimer.mark(PERFORMANCE_MARKS.total, 'start')
-    const AppConfig = getAppConfig()
-    // Get the application config which should have been stored at this point.
```

Comment:
Do we want to preserve this comment?

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/7e70ce597bbcec252558d800e3e8a80b5992dc64/packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js#L147

```
// Get the application config which should have been stored at this point.
const config = getConfig()
```